static/app/components/onboarding/productSelection.tsx

```diff
@@ -302,7 +303,7 @@ export function ProductSelection({
           : [...urlProducts, product]
       );
 
-      if (defaultProducts?.includes(ProductSolution.PROFILING)) {
+      if (defaultProducts.includes(ProductSolution.PROFILING)) {
```

Comment:
intended change?

Reply:
ya. it could be a separate PR though, not part of this. I'll clean it up